src/frontend/components/StudentLiveWrapper/index.tsx

```diff
@@ -116,17 +122,61 @@ export const StudentLiveWrapper: React.FC<StudentLiveWrapperProps> = ({
           />
         }
         mainElement={
-          isParticipantOnstage && jitsiLive ? (
-            <JitsiApiProvider value={undefined}>
-              <DashboardVideoLiveJitsi liveJitsi={jitsiLive} />
-            </JitsiApiProvider>
-          ) : (
-            <VideoPlayer
-              playerType={playerType}
-              timedTextTracks={[]}
-              video={video}
-            />
-          )
+          <PictureInPictureLayer
+            mainElement={
+              isParticipantOnstage && jitsiLive ? (
+                <DashboardVideoLiveJitsi liveJitsi={jitsiLive} />
+              ) : (
+                <VideoPlayer
+                  playerType={playerType}
+                  timedTextTracks={[]}
+                  video={video}
+                />
+              )
+            }
+            secondElement={
+              video.active_shared_live_media &&
+              (video.active_shared_live_media.urls ? (
+                <SharedMediaExplorer
+                  initialPage={video.active_shared_live_media_page!}
+                  pages={video.active_shared_live_media.urls.pages}
+                />
```

Comment:
You also need a child component that render nothing but listen to video change to update the current page

Reply:
But when the video will be updated the component will be re-render and the current page updated no ?

Reply:
no, it is store in a state